ladybug_comfort/map/utci.py

```diff
@@ -0,0 +1,118 @@
+# coding=utf-8
+"""Utility functions for calculating Universal Thermal Climate Index (UTCI).
+
+This module is devoted to calculating UTCI with NumPy.
+"""
+import numpy as np
+from ..utci import _utci_polynomial_function
+
+
+def universal_thermal_climate_index_np(ta, tr, vel, rh):
+    """Calculate Universal Thermal Climate Index (UTCI) using a polynomial approximation.
+
+    This function here is a Python version of the original UTCI_approx
+    application written in Fortran. Version a 0.002, October 2009
+    The original Fortran code can be found at www.utci.org.
+
+    Note:
+        [1] Peter Bröde, Dusan Fiala, Krzysztof Blazejczyk, Yoram Epstein,
+        Ingvar Holmér, Gerd Jendritzky, Bernhard Kampmann, Mark Richards,
+        Hannu Rintamäki, Avraham Shitzer, George Havenith. 2009.
+        Calculating UTCI Equivalent Temperature. In: JW Castellani & TL
+        Endrusick, eds. Proceedings of the 13th International Conference
+        on Environmental Ergonomics, USARIEM, Natick, MA.
+
+    Args:
+        ta: Air temperature [C] as a NumPy array.
+        tr: Mean radiant temperature [C] as a NumPy array.
+        vel: Wind speed 10 m above ground level [m/s] as a NumPy array.
+            Note that this meteorological speed at 10 m is simply 1.5 times the
+            speed felt at ground in the original Fiala model used to build UTCI.
+        rh: Relative humidity [%] as a NumPy array.
+
+    Returns:
+        UTCI_approx -- The Universal Thermal Climate Index (UTCI) for the input
+        conditions as approximated by a 4-D polynomial.
+    """
+    # set upper and lower limits of air velocity according to Fiala model scenarios
+    vel = np.where(vel < 0.5, 0.5, np.where(vel > 17, 17, vel))
+
+    # metrics derived from the inputs used in the polynomial equation
+    eh_pa = saturated_vapor_pressure_hpa(ta) * (rh / 100.0).astype(np.float32)  # partial vapor pressure
+    pa_pr = eh_pa / 10.0  # convert vapour pressure to kPa
+    d_tr = tr - ta  # difference between radiant and air temperature
+
+    # pre-calculate powers so we can re-use them
+    utci_approx = _utci_polynomial_function(ta, d_tr, vel, pa_pr)
+
+    return utci_approx
+
+
+def saturated_vapor_pressure_hpa(db_temp):
```

Comment:
Can we add `_np` to the ends of all these functions just to prevent us from confusing them with the base versions of the functions?